tests/test_points.py

```diff
@@ -84,3 +84,66 @@ def test_NoToken(self):
     def test_InvalidToken(self):
         response = self.get(self.VALID_URL, self.bogus_token())
         self.assert_invalid_token(response)
+
+
+class TestPointsPost(EndpointTestCase):
+    """
+    Test for '/api/students/<int:student_id>/points/' POST endpoint.
+    """
+
+    # Fixture specific URL to available student data.
+    VALID_URL = "/api/students/2/points/"
+    # Fixture specific URL to data not available for current user.
+    NOT_PERMITTED_URL = "/api/students/1/points/"
+    # Fixture specific URL to invalid student ID.
+    NOT_FOUND_URL = "/api/students/12345/points/"
+    # Fixture specific valid data to post a prize.
+    VALID_POST_PRIZE = {"content_type": "prize", "object_id": "2"}
+    # Fixture specific valid data to post a task.
+    VALID_POST_TASK = {"content_type": "task", "object_id": "2"}
+    # Fixture specific data to post an invalid content type.
+    INVALID_POST_TYPE = {"content_type": "asdf", "object_id": "1"}
+
+    def test_Prize_Success(self):
+        response = self.post(self.VALID_URL, self.VALID_POST_PRIZE)
```

Comment:
I just noticed, that posting prize and task modifies student "total_points". This should also be checked.